Clear up language around different orb types (#6487)
circleci/circleci-docs: jekyll/_cci2/code-coverage.md

### Codecov
{: #codecov }

Codecov has an [orb](https://circleci.com/developer/orbs/orb/codecov/codecov) to help simplify the process of uploading your coverage reports. 

> **Note:** The Codecov orb is a Partner orb, and so uncertified orbs need to be alled for your organisation in order to use it. This setting is available at **Organization Settings > Security** in the CircleCI web app.

Author: ```suggestion
**Note:** The Codecov orb is a Partner orb. You or your organization admin will need to opt-in to using uncertified orbs in order to use it. This setting is available at **Organization Settings > Security** in the CircleCI web app.
```

> **Note:** The Codecov orb is a Partner orb, and so uncertified orbs need to be alled for your organisation in order to use it. This setting is available at **Organization Settings > Security** in the CircleCI web app.

Reviewer: Do we want to capitalize "Partner"? I know we seem to be capitalizing Partner and Community, but since we don't capitalize our features, should we be capitalizing these words?

Author: no probably not, will change!

> **Note:** The Codecov orb is a Partner orb. You or your organization admin will need to opt-in to using uncertified orbs in order to use it. This setting is available at **Organization Settings > Security** in the CircleCI web app.

Reviewer 2: Does opt-in need a hyphen here?

> **Note:** The Codecov orb is a Partner orb. You or your organization admin will need to opt-in to using uncertified orbs in order to use it. This setting is available at **Organization Settings > Security** in the CircleCI web app.

Author: ```suggestion
**Note:** The Codecov orb is a Partner orb. You or your organization admin will need to opt in to using uncertified orbs in order to use it. This setting is available at **Organization Settings > Security** in the CircleCI web app.
```
